JoyboyCommunity/src/screens/Tips/index.tsx

```diff
@@ -52,8 +52,11 @@ export const Tips: React.FC = () => {
       return event.rawEvent();
     };
 
-    const feeResult = await estimateClaim.mutateAsync(await getNostrEvent(BigInt(1)));
-    const fee = BigInt(feeResult.data.fee);
+    /** @TODO add more check for the two claim flow */
+    // Default min to 0 => Change to a constant or UI estimate base claim
+    // Nonce first account tx => deduced it never do a tx before.
+    const minAmountBalanceToClaim = BigInt(0);
+    let nonce: undefined | string;
```

Comment:
No need for nonce check. It is useless. Either the account has balance, or not.

Reply:
The nonce let us check if user have no funds because it's a new account (0 tx) or no